pkgs/development/python-modules/splunk-sdk/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "splunk-sdk";
+  version = "2.1.0";
```

Comment:
```suggestion
  version = "2.1.0";
  pyproject = true;
```

Reply:
As this isn't a pyproject, can ignore this suggestion?

Reply:
While `buildPythonPackage` will still fall back to setuptools when neither is present, it's recommend to set `format` or `pyproject` ([preferred](https://github.com/NixOS/nixpkgs/issues/253154))

So technically yes, but I would highly recommend adding `pyproject = true` here regardless

Reply:
Done :)